waltid-wallet-api/src/main/kotlin/id/walt/webwallet/utils/JsonUtils.kt

```diff
@@ -1,7 +1,6 @@
 package id.walt.webwallet.utils
 
 import id.walt.crypto.utils.JsonUtils.toJsonElement
-import kotlinx.serialization.ExperimentalSerializationApi
```

Comment:
is it ok to use experimental api's (we do use them for libs, but still)?
cc @philpotisk @taminobaumann